code/modules/cargo/packs.dm

```diff
@@ -1644,7 +1643,6 @@
 	name = "Excellent Meat Crate"
 	desc = "The best cuts in the whole galaxy."
 	cost = 2000
-	var/num_contained = 10
```

Comment:
meep for this one you could have kept the var since it's used for subtypes to edit.

(this does add 5 more things of meat but that's ok tbh)